var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -138,6 +155,8 @@ class Sundials(CMakePackage, CudaPackage, ROCmPackage):
     conflicts('+trilinos',      when='@:4.1.0')
     conflicts('+monitoring',    when='@:5.5.0')
     conflicts('+rocm',          when='@:5.6.0')
+    conflicts('+profiling',     when='@:6.0.0')
+    conflicts('+caliper',       when='@:6.0.0')
```

Comment:
Since v0.17.0 variant can be conditional, so there's no need for a conflict.

Reply:
Nice! I have replaced these with the conditional variants.